packages/table/test/tableTests.tsx

```diff
@@ -833,6 +882,7 @@ describe("<Table>", () => {
 
             const viewportRect = table.state("viewportRect");
             expect(viewportRect.left).to.equal((NUM_COLS * UPDATED_COL_WIDTH) - viewportRect.width);
+            expect(onVisibleCellsChange.callCount).to.be.at.least(3);
```

Comment:
I didn't make it precise, because the callback is invoked kind of all over the place, and I simply didn't know off the top of my head how many times to expect it to be called. I'll just count the calls and tighten the bounds here though.